packages/angular/src/app/pages/crm-contact-list/user-new-form/user-new-form.component.html

```diff
@@ -0,0 +1,101 @@
+<dx-form 
+        class="plain-styled-form" 
+        labelMode="floating" 
+        [class.view-mode]="!isEditing"
```

Comment:
if isEditing is always true. We don't need to not use it here in component

Reply:
Fixed